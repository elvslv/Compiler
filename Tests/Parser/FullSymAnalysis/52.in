var i, j, k, f: integer;
r: real;
procedure proc(m, n: integer);
procedure int(i: integer);
begin
	i := 5;
end;
begin
	m := 10;
end;
begin
	if (proc(10, 9))
		i := 10;
end.