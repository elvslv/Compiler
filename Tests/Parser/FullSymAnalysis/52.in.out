
11 4 Embedded procedures are not allowed